CHANGELOG/CHANGELOG-0.1.md

```diff
@@ -0,0 +1,35 @@
+# Changelog 0.1
+
+## 0.1.0 - 2022-06-16
+
+This is the first Theia release and it includes every Antrea flow visibility capability released as of 1.6.
+
+### Added
+
+- Helm support for Theia installation. ([#15](https://github.com/antrea-io/theia/pull/15), [@yanjunz97]) ([#31](https://github.com/antrea-io/theia/pull/31), [@yanjunz97]) ([#34](https://github.com/antrea-io/theia/pull/34), [@yanjunz97])
+  * Provide Helm charts for Theia components (Spark Operator, Clickhouse, and Grafana)
+  * Allow for deploying Clickhouse with a persistent volume storage
+- Grafana dashboard: add a panel for visualizing denied connections. ([#12](https://github.com/antrea-io/theia/pull/12), [@heanlan])
+  * Introduce chord diagram for visualizing connections denied by network policies
+  * Reorganized Network policy dashboard to visualize allowed, denied, and unprotected traffic
+- Policy recommendation via Spark job. ([#16](https://github.com/antrea-io/theia/pull/16), [@dreamtalen])
+  * Add a spark job to perform network policy recommendation based on flow data pushed by Flow Aggregator
+- Theia Command Line Interface. ([#25](https://github.com/antrea-io/theia/pull/25), [@dreamtalen] [@yanjunz97] [@annakhm])
+  * Add `theia` CLI tool to provide access to Theia network flow visibility capabilities
+  * Introduce commands for starting Network Policy Recommendation, check job status, and retrieve results
+- Initial Theia Documentation. ([#40](https://github.com/antrea-io/theia/pull/45/), [@dreamtalen]) ([#45](https://github.com/antrea-io/theia/pull/45/), [@jianjuns])
+- Add Flow Agregator e2e tests for Theia. ([#14](https://github.com/antrea-io/theia/pull/14), [@wsquan171])
+- Add basic Network Policy Recommendation e2e tests. ([#27](https://github.com/antrea-io/theia/pull/27), [@heanlan])
+- Add Jenkins e2e jobs. ([#20](https://github.com/antrea-io/theia/pull/20), [@hangyan])
```

Comment:
It may be not necessary to add changes about tests here as they are not useful to users, but I'm fine to keep them if you have reached an agreement on this.

Reply:
I've added them because they are sometimes listed in Antrea changelog, but I do think they do not belong to a changelog.  Removed.

Reply:
Yes, they were sometimes added before, we no longer add them in recent releases.